doc/rst/language/spec/conversions.rst

```diff
@@ -276,6 +397,48 @@ Explicit conversions require a cast in the code. Casts are defined
 in :ref:`Casts`. Explicit conversions are supported between more
 types than implicit conversions, but not between all types.
```

Comment:
I propose the following structure of this section:
```
[The original first paragraph, unchanged.]

The supported explicit conversions are defined as follows.
[Or:] The following explicit conversions are supported [or "allowed"?].

[subsection:] Identity Conversions

An explicit conversion from a
type to the same type is allowed for any type. Such conversion does not
change the value of the expression.

[subsection:] Implicit Conversions

If an implicit conversion from the source to the target type
is allowed, a corresponding explicit conversion is allowed
as well.

[more details about implicit conversions as needed]

[subsection:] Explicit Numeric Conversions

[the existing text continues here]

[subsection:] User-defined Conversions

An explicit conversion can be implemented by ``operator :``
[continue your new text]
```

Reply:
Given that different sets of implicit conversions are allowed in different contexts, in the above statement "If an implicit conversion from the source to the target type is allowed, a corresponding explicit conversion is allowed as well." we need to say which set of implicit conversions we are talking about.